1.4/Defs/Scenarios/Scenarios_Sedentary.xml

```diff
@@ -26,13 +26,14 @@
           <method>Standing</method>
         </li>	  
 		
-	  <li Class="ScenPart_ForcedTrait">
+		<!-- The forced trait is now taken care of in the faction/pawn definition -->
+	  <!-- li Class="ScenPart_ForcedTrait">
 		<def>ForcedTrait</def>
 		<chance>1</chance>
-		<trait>NP_Nomadism_Settler</trait>
+		<trait>NP_Nomadism_Sedentary</trait>
```

Comment:
revert to original trait name to remain downwards-compatible in case the comment goes back in, or consider removing the block altogether..